### DISCUSSION TITLE
Particles

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/extensions#1679] [Ideas]
Hello, I would like to know if there was not a TurboWarp extension that allows to create particles around a sprite or clone (example: water particles, smoke, regeneration, obtain in store ...) without using the pen extension. Thanks in advance.

**Comment by CubesterYT** [comment]
You could just make particles using clones, and I think there might be an ancient extension made by SharkPool that does particles, I wouldn't recommend using extensions that aren't here in this repository, though.

@SharkPool-SP your move.

**Comment by Eaielectronic** [comment]
There are already a lot of clones in my program, so I'm looking for a way to create particles without clones.

**Comment by Eaielectronic** [comment]
And what Sharkpools extension are you talking about?

**Reply by SharkPool-SP** [reply]
> And what Sharkpools extension are you talking about?

Particle Tools, it's ancient, but it requires clones to work

**Comment by SharkPool-SP** [comment]
> Hello, I would like to know if there was not a TurboWarp extension that allows to create particles around a sprite or clone (example: water particles, smoke, regeneration, obtain in store ...) without using the pen extension. Thanks in advance.

heres a better one with no clones:
https://github.com/TurboWarp/extensions/pull/1724
